src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/DocumentResourceAPITest.java

```diff
@@ -619,4 +567,41 @@ void withTypeAndSubTypeSuppliedOnlyDocumentsInThatTypeThatMatchTheSubTypeWillBeR
             }
         }
     }
+
+    private void assertMultiDocuments(OffenderDocuments offenderDocuments) {
+
+        assertThat(offenderDocuments.getDocuments()).hasSize(7);
+        assertThat(offenderDocuments.getDocuments()).extracting(OffenderDocumentDetail::getId)
+            .doesNotContain("4191easd-6e03-4bd8-b52d-9e050eefa745");
```

Comment:
This is the ID of the document I added in the new SQL. This line and the line which asserts all the IDs which should be present are now.